### DISCUSSION TITLE
How to combine snap with mintpy?

### DISCUSSION THREAD

**Discussion by yang162132** [insarlab/MintPy#1205] [Q&A] [answered]
![image](https://github.com/insarlab/MintPy/assets/71556279/6f7f1467-1241-438b-8d33-8ff96369763c)
I have seen the documentation demo data, But I was wondering how do I process SLC data into this structure,I can only get the result data by snap

![image](https://github.com/insarlab/MintPy/assets/71556279/7bcd53bb-56f6-41a7-afe1-eb1792e93e01)

**Comment by welcome** [comment]
👋  Thanks for opening your first issue here! Please filled out the template with as much details as possible. We appreciate that you took the time to contribute!
Make sure you read our [contributing guidelines](https://github.com/insarlab/MintPy/blob/master/docs/CONTRIBUTING.md).

**Comment by yunjunz** [comment, marked as answer]
Here is brief documentation on the workflow: https://github.com/insarlab/MintPy/wiki/SNAP-input-data

**Reply by yang162132** [reply]
I've tried the process
<img width="743" alt="1719501834334" src="https://github.com/insarlab/MintPy/assets/71556279/e652f9ef-23d2-48c7-8055-697a47cc9764">
and then 
```
SNAPHU Export
SNAPHU Unwrapping
SNAPHU Import
phase to elevation
Range-Doppler Terrain Correction
Export each band to the BEAM-DIMAP file
```
Finally, get this file directory 
![image](https://github.com/insarlab/MintPy/assets/71556279/7a85afda-f079-4a19-b8cf-0a0b7d3e8287)
![image](https://github.com/insarlab/MintPy/assets/71556279/ec305b1b-9e0c-459a-a287-8ba2057906ee)
![image](https://github.com/insarlab/MintPy/assets/71556279/a263b2bb-103f-4bae-b00b-ea196a106345)
![image](https://github.com/insarlab/MintPy/assets/71556279/6eb1d253-0937-4014-bf91-e066fbbabedf)



But When I try smallbaselineApp.py, it looks like something's wrong.
![image](https://github.com/insarlab/MintPy/assets/71556279/0f62dbb0-7c75-4b96-acae-4f099e109a52)
What's wrong with my steps
